go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/iotexproject/go-pkgs v0.1.1-0.20190708233003-85a24189bbd4
 	github.com/iotexproject/iotex-address v0.2.0
 	github.com/iotexproject/iotex-election v0.1.10
-	github.com/iotexproject/iotex-proto v0.2.1-0.20190717000031-25b6ccc65ebd // indirect
+	github.com/iotexproject/iotex-proto v0.2.1-0.20190717000031-25b6ccc65ebd
```

Comment:
Why proto change?

Reply:
I have no idea. Go mod did that automatically.

Reply:
When I run `make ioctl`